### PR TITLE
Tests for form validation and error checks

### PR DESCRIPTION
**Pull request by dsesami**
##### SUMMARY
New nightwatch tests for form validation checks. Please feel free to offer commentary on formatting, style, etc.

##### ISSUE TYPE
 - Feature Pull Request

##### COMPONENT NAME
 - UI

##### AWX VERSION
devel

### CODE REVIEW

File: awx/ui/test/e2e/tests/test-form-error-handling.js [hunk 1]
```diff
@@ -0,0 +1,202 @@
...
+            .then(([jt, jt2]) => {
+                data = { jt, jt2 };
+                done();
+            });
+        client
+            .login()
+            .waitForAngular()
+            .resizeWindow(1200, 1000)
+            .useXpath()
+            .findThenClick(templatesNavTab)
```
Comment (unlikelyzero) [inline awx/ui/test/e2e/tests/test-form-error-handling.js:33]: As a general rule, I prefer to navigate by URL to the page when possible. Given how little coverage we have, this is great at catching any general regressions within the implicit navigation action here.

File: awx/ui/test/e2e/tests/test-form-error-handling.js [hunk 2]
```diff
@@ -0,0 +1,202 @@
...
+                ['invalid inventory name', client.Keys.ENTER]
+            )
+            .clearValue('//input[@name="project_name"]')
+            .setValue(
+                '//input[@name="project_name"]',
+                ['invalid project', client.Keys.ENTER]
+            );
+
+        client.expect.element('//*[@id="job_template-inventory-notfound-error"]')
+            .to.be.visible.after(3000);
```
Comment (unlikelyzero) [inline awx/ui/test/e2e/tests/test-form-error-handling.js]: we might want to consider using the explicit wait parameter if we find ourselves building this logic on many testsuites

**Review by unlikelyzero — APPROVED** [review]
Looks amazing. Just make sure that this testsuite can run against an instance 5x in a row and that it doesn't impact the results of the full e2e testrun.

**Comment by dsesami** [comment]
recheck

**Pull request merged**